DWS/Helpers/WhiteSpace.php

```diff
@@ -16,7 +16,7 @@
  * @package DWS
  * @subpackage Helpers
  */
-final class Whitespace
+final class WhiteSpace
```

Comment:
I see why this was changed to support PSR4 but shouldn't all the other references be updated as well?

Reply:
Nevermind I see they all are already the correct format. I my first look was looking at the wrong thing.